templates/components/common/navigation.html

```diff
@@ -105,4 +105,5 @@
     <div class="dropdown dropdown--quickSearch" id="quickSearch" aria-hidden="true" tabindex="-1" data-prevent-quick-search-close>
         {{> components/common/quick-search name='nav-quick-search'}}
     </div>
+    {{> components/search/quick-results-aria-message }}
```

Comment:
Could we move this to quick-search file?

Reply:
When input in focus, browser determines its wrapper as group, and reading all its content including search notification text